docs/what-is-zarf.md

```diff
@@ -0,0 +1,225 @@
+# What is Zarf?
+
+<img align="right" alt="zarf logo" src="../.images/zarf-logo.png" width="160" />
+
+Zarf was created to _**support the declarative creation & distribution of software "packages" into remote / constrained / independent Kubernetes systems**_.
+
+> "Zarf is a tool to help deploy modern stacks into air gapped environments; it's all about moving the bits." &mdash; Jeff
+
+\* Check out our [term list](#what-we-mean-by) below for clarity on all those fancy words.
+
+&nbsp;
+
+## How it helps
+
+Zarf supercharges the way you deliver complex, k8s-native applications to remote systems by reducing update preparation & installation to a few simple terminal commands. This support comes in two parts:
+
+A precompiled `zarf` binary, which:
+
+- rolls numerous software types / updates into a single distributable package (while on an internet-accessible network), and
+
+- unrolls package contents "into place" for use on production systems (while on an internet-isolated network).
```

Comment:
If we really want to sell how much Zarf `supercharges` deployments I think we could add a bit of context of how much easier the deployment of packages onto production systems is compared to the other ways people typically deploy.

Reply:
> Zarf supercharges the way you deliver complex, k8s-native applications to remote systems by reducing update preparation & installation to a few simple terminal commands.

...becomes...

> Zarf supercharges the way you deliver complex, k8s-native applications to remote systems by reducing _tens/hundreds of individual software updates, movements, and manual installations to a few simple terminal commands._

Like that?  Problem is that it's really difficult to talk much about "other ways people typically deploy" without specifying / assuming how they work.  Best I could think to do was to compare the stuff **Zarf does** against what you would have to do if you wanted to do **Zarf's work manually**.  :thinking: